package rpcz;

message rpc_request_header {
  optional int64 event_id = 1;
  // optional int32 OBSOLETE_deadline = 2;
  optional string service = 3;
  optional string method = 4;
}

message rpc_response_header {
  // optional status_code OBSOLETE_status = 1 [default = OK];
  optional int32 error_code = 2;
  optional string error_str = 3;
}
